src/components/Message/MessageSimple.tsx

```diff
@@ -211,10 +216,12 @@ const MessageSimpleWithContext = <
                 </span>
               )}
               <MessageTimestamp customClass='str-chat__message-simple-timestamp' />
-              {isEdited && (
+              {isEdited && !message.ai_generated && (
```

Comment:
Why do we need to check for `!message.ai_generated`? Is it possible a message would be edited and AI generated?

Reply:
As discussed, will move this to the `isEdited` check function.